src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1832,4 +1826,25 @@ k
         this.canvas.panY = round(UiController.km_to_px(y));
         this.dirty = true;
     };
+
+    /**
+     * Change theme to the specified name
+     *
+     * This should ONLY be called through the EventBus during a `SET_THEME` event,
+     * thus ensuring that the same theme is always in use by all app components.
+     *
+     * This method must remain an arrow function in order to preserve the scope
+     * of `this`, since it is being invoked by an EventBus callback.
+     *
+     * @for CanvasController
+     * @method _setTheme
+     * @param themeName {string}
+     */
+    _setTheme = (themeName) => {
+        if (!_has(THEME, themeName)) {
+            return;
```

Comment:
what kind of feedback should happen here and who is the target audience for this `return`?  Should this `throw` and break things hard or should this be a `console.error()` and let the developer know they aren't doing things correctly.

I feel there should be some sort of feedback here instead of just a return.